tools/aliveness/src/main.rs

```diff
@@ -0,0 +1,353 @@
+use std::{
+    collections::HashMap,
+    env,
+    mem::size_of,
+    net::{Ipv4Addr, SocketAddr, SocketAddrV4},
+    ptr::read,
+    time::Duration,
+};
+
+use aliveness::{
+    service_manager::SystemServices, BeaconResponse, BEACON_HEADER, BEACON_MULTICAST_GROUP,
+    BEACON_PORT,
+};
+use color_eyre::eyre::{bail, eyre, ContextCompat, Result, WrapErr};
+use futures_util::stream::StreamExt;
+use hula_types::{Battery, RobotState};
+use log::{error, info};
+use tokio::{
+    io::AsyncReadExt,
+    net::{UdpSocket, UnixStream},
+    select,
+    sync::{mpsc, oneshot},
+    task::JoinHandle,
+    time::interval,
+};
+use tokio_util::sync::CancellationToken;
+use zbus::{
+    zvariant::{self, OwnedObjectPath},
+    Connection, MatchRule, MessageStream, MessageType, Proxy,
+};
+
+const HULA_SOCKET_PATH: &str = "/tmp/hula";
+const HULA_RETRY_TIMEOUT: Duration = Duration::from_secs(5);
+const OS_RELEASE_PATH: &str = "/etc/os-release";
+
+async fn load_hulks_os_version() -> Result<String> {
+    let mut os_release = configparser::ini::Ini::new();
```

Comment:
As discussed in a past Dev-Meeting: Please always try to replace qualified paths with uses. Your IDE tells you the path, no need to make your code longer. Only if you have name conflicts use a longer path or alias. Recheck your code about this.

Reply:
Thanks, fixed.